File: JAVA/1502. Can Make Arithmetic Progression From Sequence.java
class Solution {
    public boolean canMakeArithmeticProgression(int[] a) {
        if(a.length==2)
        return true;
        Arrays.sort(a);
        int d=a[0]-a[1];
        for(int i=1;i<a.length-1;i++)
        {
            if(a[i]-a[i+1]!=d)
            return false;
        }
        return true;
    }
}

File: C++/1502. Can Make Arithmetic Progression From Sequence.cpp
class Solution {
public:
    bool canMakeArithmeticProgression(vector<int>& a) {
        if(a.size()==2)
        return true;
        sort(a.begin(),a.end());
        int i;
        for(i=1;i<a.size()-1;i++)
        {
            if(a[i]-a[i+1]!=a[i-1]-a[i])
            {
                return false;
                break;
            }
        }
        return true;
    }
};